Source/WebCore/platform/win/MIMETypeRegistryWin.cpp

```diff
@@ -87,6 +87,7 @@ String MIMETypeRegistry::mimeTypeForExtension(StringView string)
         mimetypeMap.add("xhtml"_s, "application/xhtml+xml"_s);
         mimetypeMap.add("rss"_s, "application/rss+xml"_s);
         mimetypeMap.add("webarchive"_s, "application/x-webarchive"_s);
+        mimetypeMap.add("avif"_s, "image/avif"_s);
```

Comment:
This should be guarded with a `USE(AVIF)`.

Reply:
JPEG XL is also guarded (I wonder why WebP is missing here)

https://github.com/WebKit/WebKit/blob/45e64ff33376b8df1116d92a6abe15ede376731e/Source/WebCore/platform/win/MIMETypeRegistryWin.cpp#L104-L106